resemble.js

```diff
@@ -478,6 +478,16 @@ var _this = {};
 					if(hasMethod) { param(); }
 					return self;
 				},
+				setTolerance: function(value){
+
+					tolerance.red = value;
+					tolerance.green = value;
+					tolerance.blue = value;
+					tolerance.alpha = value;
+
```

Comment:
Should this also set `tolerance.minBrightness`, `tolerance.maxBrightness`, `ignoreAntialiasing`, and `ignoreColors` to some specific values?